labs/jack-in/README.md

```diff
@@ -0,0 +1,18 @@
+# Jack-in
+_your experimental terminal to connect to  matrix via sliding sync_
+
+A simple example client, using sliding sync to [jack in](https://matrix.fandom.com/wiki/Jacking_in) to matrix via the sliding-sync-proxy. 
+
+## Use:
+
+You will need a [locally running sliding sync proxy]() for now.
+
+### Get the access token
+
+1. Login with a fresh session on [element.io](https://develop.element.org)
+2. verify the session against your existing session via key-cross-signing
+3. navigate to `Settings` -> `Help & About`, under _Advanced_ (on the bottom) you can find your Access token
```

Comment:
Let's not suggest this, plenty people fall into this trap and end up with a broken E2EE setup. If they want an access token, they can use `curl`.

Reply:
these were the instructions given to me for the sliding-sync-proxy and I just wrote them down so I can look it up once I forgot. Pretty certain they should not go into `main` like this and if you have better instructions, feel free to update and push them right there.